docs/shared/tutorials/react-standalone.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: 'React Standalone Tutorial'
+title: 'React App Tutorial'
 description: In this tutorial you'll create a frontend-focused workspace with Nx.
 ---
 
-# Building React Apps with the Nx Standalone Setup
+# Building a Single React App with Nx
 
 In this tutorial you'll learn how to use React with Nx in a ["standalone" (non-monorepo) setup](/concepts/integrated-vs-package-based#standalone-applications).
 
```

Comment:
Instead of `how to create a new React application`, maybe `how to add Nx to a React + Vite project`? Users should already know how to create React apps, and we're teaching them how to add Nx.